R/SVG.R

```diff
@@ -9,6 +9,12 @@
 #' @param pointsize Default point size.
 #' @param standalone Produce a standalone svg file? If \code{FALSE}, omits
 #'   xml header and default namespace.
+#' @param font_aliases List of font families to be aliased. By
+#'   default, \code{sans} is aliased to "Arial", \code{serif} to
+#'   "Times" or "Times New Roman" (the latter on Windows), and
+#'   \code{mono} to "courier".
+#' @param font_extra List of fonts or collection of fonts. These are
```

Comment:
Can you expand on the difference between `font_aliases` and `font_extras` - I don't get the difference from the docs.

A couple of examples would also be useful